R/pkg/R/context.R

```diff
@@ -345,7 +351,7 @@ spark.getSparkFilesRootDirectory <- function() {
 #'}
 #' @note spark.getSparkFiles since 2.1.0
 spark.getSparkFiles <- function(fileName) {
-  callJStatic("org.apache.spark.SparkFiles", "get", as.character(fileName))
+  file.path(spark.getSparkFilesRootDirectory(), as.character(fileName))
```

Comment:
perhaps to check, does getSparkFilesRootDirectory returns something that file.path can handle, for instance, `file://` wouldn't work properly